lookup.go

```diff
@@ -29,13 +31,39 @@ func toPeerInfos(ps []peer.ID) []*pstore.PeerInfo {
 	return out
 }
 
+func tryFormatLoggableKey(k string) (string, error) {
+	if len(k) == 0 {
+		return "", fmt.Errorf("loggableKey is empty")
+	}
+	var proto, cstr string
+	if k[0] == '/' {
```

Comment:
Will this check fail if we ever assign a multihash to `0x2F`?

Reply:
No, it'll only fail if we use 0x2F as a CID version. If this is a CID, it'll either start with the SHA256 multicodec (CIDv0) or the CID version number).

But yes, we should write these conflict rules down somewhere. Or, even better, actually *fix* this by migrating to namespaced provider records (the only case where we use raw CIDs instead of `/proto/CID`).